src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -45,4 +48,31 @@ public function helpAction(): Response
 
         return new Response();
     }
+
+    /**
+     * @Route("/comite/autocompletion",
+     *     name="app_citizen_project_committee_autocomplete",
+     *     condition="request.isXmlHttpRequest()"
+     * )
+     * @Method("GET")
+     * @Security("is_granted('IS_AUTHENTICATED_FULLY')")
+     */
+    public function committeeAutocompleteAction(Request $request)
+    {
+        $committees = $this
```

Comment:
Don't search if the term is empty, you can check this in the `condition` of the route annotation.

Reply:
✔️